### PR TITLE
add usage draft to asf_tools readme

### PR DESCRIPTION
**Pull request by Jlrine2**

### CODE REVIEW

File: asf_tools/README.md [hunk 1]
```diff
@@ -28,7 +28,23 @@ python -m pip install asf_tools
 
 ## Quick Usage
 
-TBD
+### Local Resolution Weighted Composite
+
+The `make_composite` tool allows you to create a local-resolution-weighted composite from a set of Sentinel-1 RTC
+products ([D. Small, 2012](https://doi.org/10.1109/IGARSS.2012.6350465)). It is intended to be used with RTC products generated by [ASF HyP3](https://hyp3-docs.asf.alaska.edu/using/vertex)
+
```
Comment (jhkennedy) [inline asf_tools/README.md:35]: missing a period at the end of this sentence

File: asf_tools/README.md [hunk 2]
```diff
@@ -28,7 +28,23 @@ python -m pip install asf_tools
...
+You will need to request RTC products using the `Include Scattering Area` option, then download and unzip them into an empty directory.
+
+
```
Comment (jhkennedy) [inline asf_tools/README.md:38]: extra newline here?

File: asf_tools/README.md [hunk 3]
```diff
@@ -28,7 +28,23 @@ python -m pip install asf_tools
...
+To generate a composite of the co-polarization image navigate to the directory containing the unzipped RTC products and run this command.
+```
+make_composite composite.tif */*VV.tif
+```
+
+To generate a composite of the cross-polarization image navigate to the directory containing the unzipped RTC products and run this command.
+```
```
Comment (jhkennedy) [inline asf_tools/README.md:45]: "cross-polarization image navigate to"
* images plural
* comma after image

"run this command." --> "run:"

File: asf_tools/README.md [hunk 4]
```diff
@@ -28,7 +28,23 @@ python -m pip install asf_tools
...
-TBD
+### Local Resolution Weighted Composite
+
+The `make_composite` tool allows you to create a local-resolution-weighted composite from a set of Sentinel-1 RTC
+products ([D. Small, 2012](https://doi.org/10.1109/IGARSS.2012.6350465)). It is intended to be used with RTC products generated by [ASF HyP3](https://hyp3-docs.asf.alaska.edu/using/vertex)
+
+You will need to request RTC products using the `Include Scattering Area` option, then download and unzip them into an empty directory.
+
+
+To generate a composite of the co-polarization image navigate to the directory containing the unzipped RTC products and run this command.
```
Comment (jhkennedy) [inline asf_tools/README.md:39]: "co-polarization image navigate to"
* images plural
* comma after image

"run this command." --> "run:"

File: setup.py [hunk 5]
```diff
@@ -51,7 +51,7 @@
 
     entry_points={
         'console_scripts': [
-            'make_composite.py = asf_tools.composite:main',
+            'make_composite = asf_tools.composite:main',
```
Comment (jhkennedy) [inline setup.py:54]: For this change, you also need to update:
* `tests/test_entrypoints.py`
* where it's called out in `CHANGELOG.md`

**Review by jhkennedy — CHANGES_REQUESTED** [review]

**Review by asjohnston-asf — APPROVED** [review]

**Review by jhkennedy — APPROVED** [review]

**Pull request merged**